src/trivia/libraries/trivia/trivia.go

```diff
@@ -58,24 +55,31 @@ problemloop:
 
 		select {
 		case <-timer.C:
-			fmt.Println()
-			break problemloop
+			gameOver(correct, readQuestions, prize.Amount)
+			return nil
 		case answer := <-answerCh:
-			if answer == p.AnswerOptionNumber {
+			answerNumber, err := strconv.ParseInt(strings.TrimSpace(answer), 10, 64)
+			if err != nil {
+				fmt.Println("(Use option numbers as your answer.)")
+			}
```

Comment:
it continues forward at this point, which means if you answer "a" it would get to the next if/else with answerNumber == 0. Is that what you want? It seems like it would be better to loop and get another answer.

Reply:
dbf398c